.github/workflows/github-cxx-qt-tests.yml

```diff
@@ -198,7 +198,7 @@ jobs:
     - name: "Configure"
       run: >-
         cmake
-        -D QT_DEFAULT_MAJOR_VERSION=${{ matrix.qt_version }}
+        -D USE_QT5=${{ matrix.qt_version == 5 }}
```

Comment:
oh neat, I didn't know you could use conditional expressions in this way in GitHub Actions